Automattic/wp-calypso
# Migration: Add checkbox to toggle selection for all users on user import screen

Fixes https://github.com/Automattic/wp-calypso/issues/88402

## Proposed Changes

* Adds "Select all users" checkbox toggle to Migration flow user import screen.

<img width="667" alt="CleanShot 2024-03-12 at 00 03 35@2x" src="https://github.com/Automattic/wp-calypso/assets/10482592/84b1c048-2808-4922-b602-12e7f7d14ffd">


## Testing Instructions



* Use live link or checkout branch and go through migration flow until you reach import user screen (`sites?hosting-flow=true`)
* Verify the checkbox is working correctly.
* Verify user invitation is working correctly.
* Try to break or find issues.

## Pre-merge Checklist
